cmake/cpu_features.cmake

```diff
@@ -29,7 +29,7 @@ if (ARCH_NATIVE)
     set (COMPILER_FLAGS "${COMPILER_FLAGS} -march=native")
 
 elseif (ARCH_AARCH64)
-    set (COMPILER_FLAGS "${COMPILER_FLAGS} -march=armv8-a+crc")
+    set (COMPILER_FLAGS "${COMPILER_FLAGS} -march=armv8-a+crc+simd+crypto+dotprod+ssbs")
```

Comment:
What CPUs will be not compatible?
There are some Cavium and APM servers from 2013...

Reply:
Raspberry Pi?

Reply:
Yes, Cortex A-72 does not support ssbs, everything else is supported. We found quite useful ssbs feature when kernel is enabled like for Graviton, however, not that significant, if somebody complains, we can drop it